src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -4,7 +4,7 @@
 import axios from "axios";
 
 const BASE_URL = `${window.TL.BASE_URL}ajax/analyses`;
-const USER_TYPE = window.PAGE.type;
+const USER_TYPE = window.PAGE?.type;
```

Comment:
Is the question mark supposed to be there?

Reply:
This is [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).  We use the babel plugin `@babel/plugin-proposal-optional-chaining` to allow it.

Reply:
Really it is me playing with new ecmascript features.